<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSingleUpgrading</class>
 <widget class="QWidget" name="WidgetSingleUpgrading">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFFFF;
border-color:#FAFDFF;
border: 1px solid #DFE3E5;
border-radius: 4px;
color:#2D3440;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLabel" name="labelIconUpgrade">
       <property name="minimumSize">
        <size>
         <width>66</width>
         <height>66</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>66</width>
         <height>66</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;
background-image: url(:/image/images2/Icon_doingSuccessed_big.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnConfirm">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>44</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>44</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
  width: 196px;
  height: 44px;
  background: #0047BB;
  border-radius: 4px;
  color: #FAFDFF;
}
QPushButton::disabled{
  background: rgb(55, 71, 95);
}</string>
       </property>
       <property name="text">
        <string>好的</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>30</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="labelDoing">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLabel" name="labetUpgradeText">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #A6AEBA;
border:none;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>300</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="2" colspan="2">
      <widget class="QLabel" name="labelUpgradeStatus">
       <property name="minimumSize">
        <size>
         <width>242</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>2420</width>
         <height>66</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #2D3440;
border:none;</string>
       </property>
       <property name="text">
        <string>升级完成！请重启控制柜</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
